pom.xml

```diff
@@ -48,8 +49,10 @@
     <testcontainers.version>1.15.2</testcontainers.version>
     <strimzi.testcontainers.version>0.20.0</strimzi.testcontainers.version>
     <wiremock.version>2.27.2</wiremock.version>
-    <apicurio-registry-utils-serde.version>1.2.2.Final</apicurio-registry-utils-serde.version>
+    <apicurio-registry-utils-serde.version>1.3.2.Final</apicurio-registry-utils-serde.version>
```

Comment:
Could be this controlled by quarkiverse-apicurio-registry-client dependency or this is independent on client?

Asking because end user experience is quite poor, users need to figure out many things on their own instead of just one dependency that just works.